modules/auxiliary/spoof/llmnr/llmnr_response.rb

```diff
@@ -206,9 +209,13 @@ def run
     while thread.alive?
       select(nil, nil, nil, 0.25)
     end
-
-    self.thread.kill
-    self.sock.close rescue nil
   end
 
+  def cleanup
+    if self.thread and self.thread.alive?
+      self.thread.kill
+      self.thread = nil
+    end
+    close_pcap
```

Comment:
Only if it called super(), but the Capture mixin doesn't have a cleanup() method, so this was a resource leak.